doc/commands/dataelement.md

```diff
@@ -0,0 +1,56 @@
+# Data Element
+
+- [Data Element](#data-element)
+	- [create](#create)
+	- [chattr](#chattr)
+	- [activate](#activate)
+	- [read](#read)
+
+## create
+
+Create ABAP DDIC Data Element.
+
+```bash
+sapcli dataelement create [--corrnr TRANSPORT] "DATA_ELEMENT_NAME" "Description" "PACKAGE_NAME"
```

Comment:
Are you sure we can create an object without any further information than "name", "description", and "package"?

Reply:
Yes, I am sure that for DTEL create action (POST) only name, description and package. The DTEL setup is being done via PUT request

Reply:
New command `defined` is introduced. Command `create` is not needed any more because it is covered by command `define`